<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1239</width>
    <height>339</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Ethernet</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QListWidget" name="mtuField">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>30</y>
      <width>161</width>
      <height>281</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="enableButton">
    <property name="geometry">
     <rect>
      <x>1110</x>
      <y>40</y>
      <width>121</width>
      <height>29</height>
     </rect>
    </property>
    <property name="text">
     <string>Enable</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="package_2">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>161</width>
      <height>29</height>
     </rect>
    </property>
    <property name="text">
     <string>Name</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="version_2">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>0</y>
      <width>121</width>
      <height>29</height>
     </rect>
    </property>
    <property name="text">
     <string>Speed</string>
    </property>
   </widget>
   <widget class="QListWidget" name="speedField">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>30</y>
      <width>121</width>
      <height>281</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="disableButton">
    <property name="geometry">
     <rect>
      <x>1110</x>
      <y>80</y>
      <width>121</width>
      <height>29</height>
     </rect>
    </property>
    <property name="text">
     <string>Disable</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="version">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>0</y>
      <width>161</width>
      <height>29</height>
     </rect>
    </property>
    <property name="text">
     <string>MTU</string>
    </property>
   </widget>
   <widget class="QPushButton" name="refreshButton">
    <property name="geometry">
     <rect>
      <x>1110</x>
      <y>0</y>
      <width>121</width>
      <height>29</height>
     </rect>
    </property>
    <property name="text">
     <string>Refresh</string>
    </property>
   </widget>
   <widget class="QListWidget" name="nameField">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>30</y>
      <width>161</width>
      <height>281</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="version_3">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>0</y>
      <width>121</width>
      <height>29</height>
     </rect>
    </property>
    <property name="text">
     <string>MAC address</string>
    </property>
   </widget>
   <widget class="QListWidget" name="macField">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>30</y>
      <width>121</width>
      <height>281</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="version_4">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>0</y>
      <width>121</width>
      <height>29</height>
     </rect>
    </property>
    <property name="text">
     <string>Tx</string>
    </property>
   </widget>
   <widget class="QListWidget" name="txField">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>30</y>
      <width>121</width>
      <height>281</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="version_5">
    <property name="geometry">
     <rect>
      <x>680</x>
      <y>0</y>
      <width>121</width>
      <height>29</height>
     </rect>
    </property>
    <property name="text">
     <string>Rx</string>
    </property>
   </widget>
   <widget class="QListWidget" name="rxField">
    <property name="geometry">
     <rect>
      <x>680</x>
      <y>30</y>
      <width>121</width>
      <height>281</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="powerButton">
    <property name="geometry">
     <rect>
      <x>1110</x>
      <y>120</y>
      <width>121</width>
      <height>29</height>
     </rect>
    </property>
    <property name="text">
     <string>Reset MAC</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="version_6">
    <property name="geometry">
     <rect>
      <x>800</x>
      <y>0</y>
      <width>121</width>
      <height>29</height>
     </rect>
    </property>
    <property name="text">
     <string>Disable status</string>
    </property>
   </widget>
   <widget class="QListWidget" name="disableField">
    <property name="geometry">
     <rect>
      <x>800</x>
      <y>30</y>
      <width>121</width>
      <height>281</height>
     </rect>
    </property>
   </widget>
   <widget class="QListWidget" name="runField">
    <property name="geometry">
     <rect>
      <x>920</x>
      <y>30</y>
      <width>121</width>
      <height>281</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="version_7">
    <property name="geometry">
     <rect>
      <x>920</x>
      <y>0</y>
      <width>121</width>
      <height>29</height>
     </rect>
    </property>
    <property name="text">
     <string>Running</string>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
